.github/workflows/c-cpp.yml

```
@@ -235,28 +244,29 @@
 
     - name: Compile for macOS
       shell: bash
+      working-directory: ${{ github.env.__srcdir }}
```

actionlint (1.7.4)

receiver of object dereference "__srcdir" must be type of object but got "string"

(expression)